pkg/gameserverallocations/controller_test.go

```diff
@@ -806,11 +807,13 @@ func createRequest(gsa *allocationv1.GameServerAllocation) (*http.Request, error
 	}
 
 	r, err := http.NewRequest(http.MethodPost, "/", buf)
-	r.Header.Set("Content-Type", k8sruntime.ContentTypeJSON)
-
 	if err != nil {
 		return nil, err
 	}
+
+	r.Header.Set("Content-Type", k8sruntime.ContentTypeJSON)
+	r.Header.Set("Accept", k8sruntime.ContentTypeJSON)
```

Comment:
Not sure we want this? Either the response is always json (even errors), and we never hit the `err = json.Unmarshal(jsn, ret)` on line 783, or we expect errors to be in a non-json format like a plain text stack trace, so we want those errors to fall through and get caught and wrapped and returned.